Assets/NuGet/Editor/NugetHelper.cs

```diff
@@ -554,12 +554,11 @@ private static bool IsAlreadyImportedInEngine(NugetPackageIdentifier package)
         private static HashSet<string> alreadyImportedLibs = null;
         private static HashSet<string> GetAlreadyImportedLibs()
         {
+            
             if (alreadyImportedLibs == null)
             {
-                string[] lookupPaths = GetAllLookupPaths();
-                IEnumerable<string> libNames = lookupPaths
-                    .SelectMany(directory => Directory.EnumerateFiles(directory, "*.dll", SearchOption.AllDirectories))
-                    .Select(Path.GetFileName)
+                IEnumerable<string> libNames = AppDomain.CurrentDomain.GetAssemblies()
```

Comment:
A little worried that this will catch assemblies we've installed with NuGet. For example, if you install a package, then try to update it, will this find the previous version and fail to update the assembly?